Remotion/Web/Development.WebTesting/CoypuElementScopeExtensions.cs

```diff
@@ -55,11 +55,11 @@ public static void EnsureExistence ([NotNull] this ElementScope scope)
       }
       catch (MissingHtmlException exception)
       {
-        throw AssertionExceptionUtility.CreateControlMissingException (exception.Message);
+        throw AssertionExceptionUtility.CreateControlMissingException (null, exception.Message);

```

Comment:
Add Driver parameter to method.

Reply:
Unfortunately, I do not know how. I only have the `ElementScope` parameter in this method availabe, from which - from what I have gathered - I am not able to retrieve the desired driver.

Reply:
We already needed/did this in `Remotion.Web.Development.WebTesting.CoypuElementScopeFillInWithAndSendKeysExtensions.GetDriver`. This API could be provided as an extension method. Not sure how to feel about this though.

```csharp
private static readonly Lazy<FieldInfo> s_driverFieldInfo = new Lazy<FieldInfo> (
    () => Assertion.IsNotNull (
        typeof (ElementScope).GetField ("_driver", BindingFlags.NonPublic | BindingFlags.Instance),
        "Coypu has changed, please update CoypuElementScopeFillInWithAndSendKeysExtensions.GetDriver() method."),
    LazyThreadSafetyMode.ExecutionAndPublication);

private static IDriver GetDriver ([NotNull] this ElementScope scope)
{
  ArgumentUtility.CheckNotNull ("scope", scope);

  return (IDriver) s_driverFieldInfo.Value.GetValue (scope);
}
```

@MichaelKetting

Reply:
> Do you think we should?

Would definitely make things easier. 

> And ading a parameter or using the DriverScope.Driver, that's almost negligent if we didn't do this. Keep it internal for now, though.

What would the parameter solution be?

Reply:
requiring that the driver be passed as a parameter instead of getting it from scope.

Reply:
for the imediate task, we can use the reflection solution and then have a follow-up task for coypu. that way, we can clsoe this PR.